pyproject.toml

```diff
@@ -63,6 +63,9 @@ sdist.include = [".git", "vendor/llama.cpp/.git"]
 provider = "scikit_build_core.metadata.regex"
 input = "llama_cpp/__init__.py"
 
+[project.scripts]
```

Comment:
Can we actually keep this out for now, the `pyhton3 -m llama_cpp.server` approach is okay for the time being and we may want to think through the name a little more for adding an executable.